main.go

```diff
@@ -162,6 +164,30 @@ func dumpJSONFile(file string, pretty bool, schema bool) error {
 	return nil
 }
 
+func getFilesContentHash(files []string) (string, error) {
+
+	var contents []byte
+	for _, f := range files {
+		c, err := os.ReadFile(f)
+		if err != nil {
+			return "", err
+		}
+		contents = append(contents, c...)
+	}
+
+	sha1 := sha1.New()
+	_, err := sha1.Write(contents)
+	if err != nil {
+		return "", err
+	}
```

Comment:
I think we can reorganize this a bit to not put the whole thing in memory all the time:

```
sha1 := sha1.New()
for _, f := range files {
  f, err := os.Open()
  // handle err

  _, err = io.Copy(f, sha1)
  // handle err
}

return hex.EncodeToString(sha1.Sum(nil)), nil
```